src/xrCore/Animation/Bone.hpp

```diff
@@ -350,23 +348,23 @@ class XRCORE_API CBone : public CBoneInstance, public IBoneData
     shared_str Name() { return name; }
     shared_str ParentName() { return parent_name; }
     shared_str WMap() { return wmap; }
-    IC CBone* Parent() { return parent; }
-    IC BOOL IsRoot() { return (parent == 0); }
-    shared_str& NameRef() { return name; }
+    [[nodiscard]] IC CBone* Parent() const { return parent; }
+    [[nodiscard]] IC BOOL IsRoot() const { return (parent == nullptr); }
+    [[maybe_unused]] shared_str& NameRef() { return name; }
     // transformation
-    const Fvector& _Offset() { return mot_offset; }
-    const Fvector& _Rotate() { return mot_rotate; }
-    float _Length() { return mot_length; }
-    IC Fmatrix& _RTransform() { return rest_transform; }
-    IC Fmatrix& _RITransform() { return rest_i_transform; }
-    IC Fmatrix& _LRTransform() { return local_rest_transform; }
-    IC Fmatrix& _MTransform() { return mot_transform; }
-    IC Fmatrix& _LTransform() { return mTransform; } //{return last_transform;}
-    IC const Fmatrix& _LTransform() const { return mTransform; }
-    IC Fmatrix& _RenderTransform() { return mRenderTransform; } //{return render_transform;}
```

Comment:
Return removed comment, but you can format it:
```
// { return render_transform; }
```

Reply:
Restored.